index.html

```diff
@@ -15,10 +16,10 @@
   </head>
   <body>
     <main>
-    <section class="parent">
-      <div>
+      <section class="parent">
+        <div>
 
-        <header>
+      <header>
         <nav>
           <a href="#">Risk</a>
         </nav>
```

Comment:
- [ ] I kindly advise that you follow the defined order of semantic tags as this is very important for google to understand your page and for accessibility in general
```
<body>
    <header>...</header>
    <main>
         <section>....</section>
   </main>
</body>
```